terraform/environments/bootstrap/member-bootstrap/instance-scheduler.tf

```diff
@@ -23,8 +23,7 @@ data "aws_iam_policy_document" "instance-scheduler-access" {
       "ec2:DescribeInstances",
       "ec2:DescribeInstanceStatus",
       "ec2:DescribeTags",
-      "ec2:StartInstances",
-      "ec2:StopInstances"
```

Comment:
Did you mean to remove the ability for the instance scheduler to stop instances?

Reply:
Edd put together this release while I was off. I assumed this is added as part of the later release when the IAM changes are made but not 100% sure